extension/data/modules/profile.js

```diff
@@ -851,6 +876,47 @@ function profilepro () {
             }
             makeProfile(user, listing, options);
         });
+
+        // Open profile bashed on hash params
+        window.addEventListener('TBHashParams', event => {
+            const listing = event.detail.tbprofile;
+            if (listing && listingTypes.includes(listing)) {
```

Comment:
Passing falsy values to `Array.includes()` is fine, so the `listing &&` doesn't seem necessary here
```suggestion
            if (listingTypes.includes(listing)) {
```